echopype/echodata/convention/echodata_structures.py

```diff
@@ -0,0 +1,327 @@
+#############################################################################################
+# PLEASE CAREFULLY REVIEW THE RULES BEFORE APPLYING CHANGES.
+#
+# Rules of the structure:
+#
+# 1. For each sensor, the keys of each group should correspond to the groups in 1.0.yml
+# 2. The ep_group should correspond to the group that would be used to access the DataTree.
+# 2. coords should contain all POSSIBLE unique coordinates.
+# 3. If you add a version structure, please add this version to all sensors.
```

Comment:
Just a note that while this statement (3) looks strictly correct, we don't anticipate ever adding other versions. This is a one time transition.

Reply:
I think the hope is that this is a one time thing, but I thought I would put it in there just in case.